src/frame-writer.cpp

```diff
@@ -10,6 +10,7 @@
 #include <cstring>
 #include "averr.h"
 
+extern std::atomic<bool> exit_main_loop;
```

Comment:
Add a comment where this comes from (main.cpp in this case).

Reply:
Or maybe it is better to add a reference to that variable in FrameWriterParams instead?